src/sentry/integrations/github_enterprise/integration.py

```diff
@@ -93,7 +93,12 @@
 )
 
 
-API_ERRORS = {404: "GitHub Enterprise returned a 404 Not Found error.", 401: ERR_UNAUTHORIZED}
+API_ERRORS = {
+    404: "If this repository exists, ensure"
+    " that your installation has permission to access this repository"
+    " (https://github.com/settings/installations).",
```

Comment:
👎 on implicit string concatenation. Just my 2c

Reply:
What if there are parenthesis? Is that easier to grok?

Reply:
Hmm, I'd also say that's a change that should probably be a different PR since its not only here that we do that.
eg. https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/constants.py#L4-L6

Reply:
I'd say we shouldn't make things worse, since we're touching code here let's be explicit

Reply:
I changed this one spot to use `+` instead, lmk if I should use parenthesis instead and/or if I should apply this to other strings as well :+1: